draco/synthesis/gain.py

```diff
@@ -19,10 +18,121 @@
 from ..core import containers, task
 
 
-class RandomGains(task.SingleTask):
-    r"""Generate a random gaussian realisation of gain timestreams.
+class BaseGains(task.SingleTask):
+    r"""Rudimentary class to generate gain timestreams.
```

Comment:
This doesn't need to be a raw string (i.e. `r"""`). That's only needed below as it contains Latex source.